Tests/UITests/Tests.cs

```diff
@@ -68,54 +70,95 @@ public void TestEnablingAndDisablingServices()
         }
 
         [Test]
-        public void TestServiceStatePersistence()
+        public void TestServiceStatePersistenceCrashes()
         {
+            app.Screenshot("App Launched - Ready for tests");
             ServiceStateHelper.app = app;
+            app.WaitForElement(TestStrings.GoToTogglePageButton);
             app.Tap(TestStrings.GoToTogglePageButton);
 
             /* Make sure Crashes enabled state is persistent */
             ServiceStateHelper.AppCenterEnabled = true;
             ServiceStateHelper.CrashesEnabled = false;
             Assert.IsFalse(ServiceStateHelper.CrashesEnabled);
+            //There seems to be some sort of timing issue here.
+            //   without the Thread.Sleep - this code fails although I see that the method called is Async (with a Wait)
+            //   so I think this deserves further investigation. I've added a terribly long wait to get the tests green.
+            Thread.Sleep(10000);
```

Comment:
I'm also skeptical that the thread need to be slept for this long.  If that's the case then there may be a different issue that we need to address.  Also I would like to align the sleep times with the Exception tests.

Reply:
@jwallra It is long but I'd rather our UITests be slow than fail intermittently since they are running over night anyways

Reply:
That was my thinking too - I can tinker with the times but I wanted to be sure that it cleared the issue (network speeds can be slow so if it's something to do with that it's difficult to say how long we need to wait for here).